content/blog/2016/2016-04-07-jenkins-community-survey-results-blog.adoc

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: "Jenkins Community Survey Results: CD, Pipelines, and Star Wars"
+tags:
+- jenkins
+- survey
+- continuous delivery
+- pipelines
+- docker
+- star wars
+author: bvdawson
+---
+
+*Jenkins Community Survey Results –Continuous Delivery, Pipelines and
+Star Wars*
+
+image:/images/blog/2016-survey-blog-butler.png[image,width=84,height=115]Last
+fall, at the Jenkins User Conference – US West (JUC), the community
```

Comment:
I disagree with this statement. I don't recall the Jenkins community curating a survey. From my understanding CloudBees crafted a survey for the Jenkins user community, which is IMO fine to say here.